compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -98,11 +98,17 @@
 #include "x/codegen/X86Instruction.hpp"
 #include "codegen/InstOpCode.hpp"
 
+// Amount to be added to the estimated code size to ensure that there are long
+// branches between warm and cold code sections (must be multiple of 8 bytes).
+//
+#define MIN_DISTANCE_BETWEEN_WARM_AND_COLD_CODE 512
+
 namespace OMR { class RegisterUsage; }
 namespace TR { class RegisterDependencyConditions; }
 
 // Hack markers
 #define CANT_REMATERIALIZE_ADDRESSES(cg) (cg->comp()->target().is64Bit()) // AMD64 produces a memref with an unassigned addressRegister
+#define OVER_ESTIMATATION 4
```

Comment:
Please add a comment describing what this means and what it is used for.

Reply:
Not needed. I will remove it.

Reply:
Done.